playbooks/provisioning/aws/provision_aws.yml

```diff
@@ -0,0 +1,16 @@
+- hosts: localhost
+  become: no
+  gather_facts: no
+  pre_tasks:
+  - debug: var=openshift_cluster_id
+  - debug: var=openshift_env_id
+
+  # TODO: properly handle these defaults, currently role defaults are overriding
+  - fail:
+      msg: "Missing required variable: openshift_cluster_id"
+    when: openshift_cluster_id is not defined
```

Comment:
I wish I had written down why I ended up choosing this route, but there were some cases where `is not defined` didn't catch things so I ended up using `when: item not in vars and item not in hostvars[inventory_hostname]`. I'll try to remember why.